Makefile

```diff
@@ -236,8 +236,11 @@ dist: clean
 	pushd dist && sha256sum * > SHA256SUMS && popd
 
 packages: dist
+	mkdir -p dist/tmp
+	unzip dist/logcli-linux-amd64.zip -d dist/tmp
```

Comment:
Will this remove the zip file again?

Reply:
I don't think so... but I can double check

Reply:
confirmed, zip is still there